src/index.html

```diff
@@ -6,13 +6,65 @@
       name="viewport"
       content="width=device-width, initial-scale=1.0"
     />
-    <title>Product cards</title>
     <link
       rel="stylesheet"
       href="./styles/index.scss"
```

Comment:
The link to the SCSS file should be to a CSS file. Browsers do not understand SCSS files directly, they must be preprocessed to CSS. Change the href attribute to point to a `.css` file instead.